<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvanceSettings</class>
 <widget class="QWidget" name="AdvanceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdvanceSetting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QWidget{
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
        <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
        </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
           <width>839</width>
           <height>776</height>
       </rect>
      </property>
         <layout class="QGridLayout">
             <item row="0" column="0" colspan="4">
                 <widget class="QWidget" name="widgetVideoPage" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_3">
                         <property name="leftMargin">
                             <number>0</number>
                         </property>
                         <property name="topMargin">
                             <number>0</number>
                         </property>
                         <property name="rightMargin">
                             <number>0</number>
                         </property>
                         <property name="bottomMargin">
                             <number>0</number>
                         </property>
                         <item>
                             <widget class="QPushButton" name="btnVideoPageIcon">
                                 <property name="sizePolicy">
                                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                         <horstretch>0</horstretch>
                                         <verstretch>0</verstretch>
                                     </sizepolicy>
                                 </property>
                                 <property name="minimumSize">
                                     <size>
                                         <width>24</width>
                                         <height>24</height>
                                     </size>
                                 </property>
                                 <property name="maximumSize">
                                     <size>
                                         <width>24</width>
                                         <height>24</height>
                                     </size>
                                 </property>
                                 <property name="styleSheet">
                                     <string notr="true">.QPushButton{
	border: none;
}</string>
                                 </property>
                                 <property name="icon">
                                     <iconset resource="../../../resource/sVanilla.qrc">
                                         <normaloff>:/icon/bar/video.svg</normaloff>:/icon/bar/video.svg
                                     </iconset>
                                 </property>
                                 <property name="iconSize">
                                     <size>
                                         <width>20</width>
                                         <height>20</height>
                                     </size>
                                 </property>
                                 <property name="_vanillaStyle_Patch" stdset="0">
                                     <string>ButtonIconPatch</string>
                                 </property>
                             </widget>
                         </item>
                         <item>
                             <widget class="QLabel" name="labelVideoWidget">
                                 <property name="font">
                                     <font>
                                         <bold>true</bold>
                                     </font>
                                 </property>
                                 <property name="text">
                                     <string>Video Page</string>
                                 </property>
                             </widget>
                         </item>
                     </layout>
                 </widget>
             </item>
             <item row="2" column="0">
                 <widget class="QLabel" name="labelDefaultWIdget">
                     <property name="text">
                         <string>Default View</string>
                     </property>
                 </widget>
             </item>
             <item row="13" column="0">
                 <widget class="QLabel" name="labelAria2Port">
                     <property name="text">
                         <string>Port</string>
                     </property>
                 </widget>
             </item>
             <item row="6" column="2">
                 <widget class="QCheckBox" name="cbxNoParseList">
                     <property name="text">
                         <string>Enable</string>
                     </property>
                 </widget>
             </item>
             <item row="8" column="0" colspan="4">
                 <widget class="QWidget" name="aria2SettingTitle" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_4">
                         <property name="leftMargin">
                             <number>0</number>
                         </property>
                         <property name="topMargin">
                             <number>0</number>
                         </property>
                         <property name="rightMargin">
                             <number>0</number>
                         </property>
                         <property name="bottomMargin">
                             <number>0</number>
                         </property>
                         <item>
                             <widget class="QPushButton" name="iconAria2Settings">
                                 <property name="sizePolicy">
                                     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                         <horstretch>0</horstretch>
                                         <verstretch>0</verstretch>
                                     </sizepolicy>
                                 </property>
                                 <property name="minimumSize">
                                     <size>
                                         <width>24</width>
                                         <height>24</height>
                                     </size>
                                 </property>
                                 <property name="maximumSize">
                                     <size>
                                         <width>24</width>
                                         <height>24</height>
                                     </size>
                                 </property>
                                 <property name="styleSheet">
                                     <string notr="true">.QPushButton{
                                         border: none;
                                         }
                                     </string>
                                 </property>
                                 <property name="icon">
                                     <iconset resource="../../../resource/sVanilla.qrc">
                                         <normaloff>:/icon/bar/download.svg</normaloff>:/icon/bar/download.svg
                                     </iconset>
                                 </property>
                                 <property name="iconSize">
                                     <size>
                                         <width>20</width>
                                         <height>20</height>
                                     </size>
                                 </property>
                                 <property name="_vanillaStyle_Patch" stdset="0">
                                     <string>ButtonIconPatch</string>
                                 </property>
                             </widget>
                         </item>
                         <item>
                             <widget class="QLabel" name="labelAria2CommonSettings">
            <property name="font">
             <font>
                 <bold>true</bold>
             </font>
            </property>
            <property name="text">
                <string>Aria2 Settings</string>
            </property>
           </widget>
          </item>
                     </layout>
                 </widget>
             </item>
             <item row="14" column="2">
                 <widget class="QLineEdit" name="lineEditAria2Token"/>
             </item>
             <item row="2" column="2">
                 <widget class="QWidget" name="widgetDefaultView" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_8">
                         <property name="leftMargin">
                             <number>0</number>
                         </property>
                         <property name="topMargin">
                             <number>0</number>
                         </property>
                         <property name="rightMargin">
                             <number>0</number>
                         </property>
                         <property name="bottomMargin">
                             <number>0</number>
                         </property>
                         <item>
                             <widget class="QCheckBox" name="cbxGrid">
                                 <property name="text">
                                     <string>Grid</string>
                                 </property>
                             </widget>
                         </item>
                         <item>
                             <widget class="QCheckBox" name="cbxList">
                                 <property name="text">
                                     <string>List</string>
                                 </property>
                             </widget>
                         </item>
                     </layout>
                 </widget>
             </item>
             <item row="15" column="2">
                 <widget class="QCheckBox" name="checkBoxEnableAria2AdvancedSettings">
                     <property name="text">
                         <string>Enable</string>
                     </property>
                 </widget>
             </item>
             <item row="2" column="3">
                 <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                         <enum>Qt::Orientation::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                         <size>
                             <width>40</width>
                             <height>20</height>
                         </size>
                     </property>
                 </spacer>
             </item>
             <item row="6" column="0">
                 <widget class="QLabel" name="label_3">
                     <property name="text">
                         <string>NoParsePlayList</string>
                     </property>
                 </widget>
             </item>
             <item row="1" column="0" colspan="4">
                 <widget class="DividingLine" name="lineVideoPage" native="true"/>
             </item>
             <item row="13" column="2">
                 <widget class="QSpinBox" name="spinBoxAria2Port">
                     <property name="maximum">
                         <number>65535</number>
                     </property>
                 </widget>
             </item>
             <item row="4" column="2">
                 <widget class="QComboBox" name="comboBoxOrderElem">
                     <item>
                         <property name="text">
                             <string>default</string>
                         </property>
                     </item>
                     <item>
                         <property name="text">
                             <string>title</string>
                         </property>
                     </item>
                     <item>
                         <property name="text">
                             <string>time</string>
                         </property>
                     </item>
                     <item>
                         <property name="text">
                             <string>duration</string>
                         </property>
                     </item>
                 </widget>
             </item>
             <item row="14" column="0">
                 <widget class="QLabel" name="labelAria2Token">
                     <property name="text">
                         <string>Token</string>
                     </property>
                 </widget>
             </item>
             <item row="4" column="0">
                 <widget class="QLabel" name="labelOrderBy">
                     <property name="text">
                         <string>Sort by</string>
                     </property>
                 </widget>
             </item>
             <item row="17" column="0" colspan="3">
                 <widget class="Aria2AdvanceSettings" name="listWidgetAria2AdvancedSettings">
                     <property name="sizePolicy">
                         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                         </sizepolicy>
                     </property>
                 </widget>
             </item>
             <item row="5" column="2">
                 <widget class="QWidget" name="widgetSortOrder" native="true">
                     <layout class="QHBoxLayout" name="horizontalLayout_10">
                         <property name="leftMargin">
                             <number>0</number>
                         </property>
                         <property name="topMargin">
                             <number>0</number>
                         </property>
                         <property name="rightMargin">
                             <number>0</number>
                         </property>
                         <property name="bottomMargin">
                             <number>0</number>
                         </property>
          <item>
              <widget class="QCheckBox" name="cbxDes">
                  <property name="text">
                      <string>Descend</string>
                  </property>
              </widget>
          </item>
                         <item>
                             <widget class="QCheckBox" name="cbxAsc">
                                 <property name="text">
                                     <string>Ascend</string>
                                 </property>
                             </widget>
                         </item>
                     </layout>
                 </widget>
             </item>
             <item row="9" column="0" colspan="4">
                 <widget class="DividingLine" name="lineAria2Setting" native="true"/>
             </item>
             <item row="3" column="0" colspan="3">
                 <widget class="QLabel" name="labelDefaultExplain">
                     <property name="sizePolicy">
                         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                         </sizepolicy>
                     </property>
                     <property name="palette">
                         <palette>
                             <active>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </active>
                             <inactive>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </inactive>
                             <disabled/>
                         </palette>
                     </property>
                     <property name="font">
                         <font>
                             <pointsize>10</pointsize>
                         </font>
                     </property>
                     <property name="text">
                         <string>The video page defaults to using a grid layout
                             where &quot;default&quot; here refers to each new search starting.
                         </string>
                     </property>
                     <property name="wordWrap">
                         <bool>true</bool>
                     </property>
                 </widget>
             </item>
             <item row="12" column="0">
                 <widget class="QLabel" name="labelAria2Url">
                     <property name="text">
                         <string>Url</string>
                     </property>
                 </widget>
             </item>
             <item row="11" column="0" colspan="3">
                 <widget class="QLabel" name="labelEnableRemoteExplain">
                     <property name="palette">
                         <palette>
                             <active>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </active>
                             <inactive>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </inactive>
                             <disabled/>
                         </palette>
                     </property>
                     <property name="font">
                         <font>
                             <pointsize>10</pointsize>
                         </font>
                     </property>
                     <property name="text">
                         <string>Configure to use remote aria2 for downloading</string>
                     </property>
                 </widget>
             </item>
             <item row="5" column="0">
                 <widget class="QLabel" name="labelSortVideoOrder">
                     <property name="text">
                         <string>Sort Order</string>
                     </property>
                 </widget>
             </item>
             <item row="16" column="0" colspan="3">
                 <widget class="QLabel" name="label">
                     <property name="palette">
                         <palette>
                             <active>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </active>
                             <inactive>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </inactive>
                             <disabled/>
                         </palette>
                     </property>
                     <property name="font">
                         <font>
                             <pointsize>10</pointsize>
                         </font>
                     </property>
                     <property name="text">
                         <string>Enable advanced settings for aria2
                             Only for advanced users who need to customize settings.
                         </string>
                     </property>
                     <property name="wordWrap">
                         <bool>true</bool>
                     </property>
                 </widget>
             </item>
             <item row="7" column="0" colspan="3">
                 <widget class="QLabel" name="labelNoParsePlayListExplain">
                     <property name="palette">
                         <palette>
                             <active>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </active>
                             <inactive>
                                 <colorrole role="WindowText">
                                     <brush brushstyle="SolidPattern">
                                         <color alpha="66">
                                             <red>0</red>
                                             <green>0</green>
                                             <blue>0</blue>
                                         </color>
                                     </brush>
                                 </colorrole>
                             </inactive>
                             <disabled/>
                         </palette>
                     </property>
                     <property name="font">
                         <font>
                             <pointsize>10</pointsize>
                         </font>
                     </property>
                     <property name="text">
                         <string>Not parsing the playlist means that the video corresponding to the given link is
                             included in a playlist And by default, the playlist is parsed. Only download the single
                             video corresponding to the link.
                         </string>
                     </property>
                     <property name="wordWrap">
                         <bool>true</bool>
                     </property>
                 </widget>
             </item>
             <item row="12" column="2">
                 <widget class="QLineEdit" name="lineEditAria2Url"/>
             </item>
             <item row="10" column="0">
                 <widget class="QLabel" name="label_2">
                     <property name="text">
                         <string>Remote Aria2</string>
                     </property>
                 </widget>
             </item>
             <item row="10" column="2">
                 <widget class="QCheckBox" name="checkBoxAria2EnableRemote">
                     <property name="text">
                         <string>Enable</string>
                     </property>
                 </widget>
             </item>
             <item row="15" column="0">
                 <widget class="QLabel" name="label_4">
                     <property name="text">
                         <string>Aria2 Advanced Settings</string>
                     </property>
                 </widget>
             </item>
             <item row="18" column="0">
                 <spacer name="verticalSpacer">
                     <property name="orientation">
                         <enum>Qt::Orientation::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                         <size>
                             <width>20</width>
                             <height>40</height>
                         </size>
                     </property>
                 </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Aria2AdvanceSettings</class>
   <extends>QListWidget</extends>
   <header>SUI/Aria2Settings/Aria2AdvanceSettings.h</header>
  </customwidget>
  <customwidget>
   <class>DividingLine</class>
   <extends>QWidget</extends>
   <header>SUI/DividingLine.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../resource/sVanilla.qrc"/>
 </resources>
 <connections/>
</ui>
